src/main/java/com/suse/salt/netapi/results/SSHRawResult.java

```diff
@@ -0,0 +1,30 @@
+package com.suse.salt.netapi.results;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Wrapper class for salt-ssh results.
+ *
+ */
+public class SSHRawResult {
+
+    private int retcode;
+    private Optional<String> stderr = Optional.empty();
+    private Optional<String> stdout = Optional.empty();
```

Comment:
i'm not sure but wasn't the raw ssh the one where stdout/stderr is always present regardless of success or failure?